templates/repository/common/ADOPTERS.md

```diff
@@ -176,6 +176,12 @@ that your company deserves a spot here, reach out to
             <td align="center"><img height="32px" src="https://raw.githubusercontent.com/ory/meta/master/static/adopters/connctd.svg" alt="Connctd"></td>
             <td><a href="https://connctd.com/">connctd.com</a></td>
         </tr>
+        <tr>
+            <td>Adopter *</td>
+            <td>Paralus</td>
+            <td align="center"><img height="32px" src="../../../static/adopters/paralus.svg" alt="Paralus"></td>
```

Comment:
Following the structure of the other entries, this should work once this PR is merged:

```suggestion
            <td align="center"><img height="32px" src="[../../../static](https://raw.githubusercontent.com/ory/meta/master/static/adopters/paralus.svg" alt="Paralus"></td>
```

Reply:
yea my bad, context switching too much and forgot to replace the test URL, 
should be fine now!